src/Features/Core/Portable/CodeCleanup/AbstractCodeCleanupService.cs

```diff
@@ -70,6 +83,13 @@ protected AbstractCodeCleanupService(ICodeFixService codeFixService)
                 }
             }
 
+            if (enabledDiagnostics.RunThirdPartyFixers)
+            {
+                document = await ApplyThirdPartyCodeFixesAsync(
+                    document, progressTracker, options, cancellationToken).ConfigureAwait(false);
+                progressTracker.ItemCompleted();
```

Comment:
can we set teh description of the progress tracker so that users know what is going on?  otherwise this will blamed against whatever ran before (like `Formatting_document`)

Reply:
so, instead of special casing our fixers and third party ones, can we just have this all be the same thing and just have a uniform system where our analyzers/fixers and 3rd party ones are basically treated the same in this entire cleanup step?

Reply:
will refactor to show the titles for our analyers as well + show more granular progress